File: DS Assignments/ProgrammingAssignment4/ProgrammingAssignment4/ProgrammingAssignment4/Surgeon.cpp
#include "Surgeon.h"

Surgeon::Surgeon()
{
	Surgeon::Doctor( " ", 0, " ");
	operating = false;
}
Surgeon::Surgeon(string empName, int empNumber, string special, bool isOp)
{
	Surgeon::Doctor(empName, empNumber, special);
	operating = isOp;
}
bool Surgeon::getIsOperating()
{
	return operating;
}
void Surgeon::setIsOperating(bool isO)
{
	operating = isO;
}
string Surgeon::toString()
{
	char OP = 'N';
	if (operating)
		OP='y';
	return Surgeon::Doctor::toString() + "Operating: "+ OP;
}

File: DS Assignments/ProgrammingAssignment1/ProgrammingAssignment1/src/programmingassignment1/Surgeon.java
// COSC 2320 Summer 2015
// Ali Elsaadi
// Progamming Assignment 1
// This is my own work; I will not post
package programmingassignment1;


public class Surgeon extends Doctor {
    
    private boolean operating;
    public Surgeon ()
    {
        super (" ", 0, " ");
        operating = false;
    }
    
    public Surgeon (String empName, int empNumber, String special, boolean isOp)
    {
        super (empName, empNumber, special);
        operating = isOp;
    }
    
    public boolean getIsOperating()
    {
        return operating;
    }
    public void setIsOperating(boolean isO)
    {
        operating = isO;
    }
    
    public String tostring()
    {
        char OP = 'N';
        if (operating)
             OP = 'Y';
        return super.toString() + "Operating: " + OP;
    }
}
